spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/SimpleJobService.java

```diff
@@ -339,62 +325,4 @@ private void checkJobExists(String jobName) throws NoSuchJobException {
 			throw new NoSuchJobException("No Job with that name either current or historic: [" + jobName + "]");
 		}
 	}
-
```

Comment:
Yeh this code was ancient. I guess it was also not tested otherwise this should have made some test fail.